python_modules/libraries/dagster-looker/dagster_looker/api/dagster_looker_api_translator.py

```diff
@@ -89,9 +104,17 @@ class LookmlView:
 class LookerStructureData:
     structure_type: LookerStructureType
     data: Union[LookmlView, LookmlModelExplore, DashboardFilter, Dashboard]
+    base_url: Optional[str] = None
 
 
 class DagsterLookerApiTranslator:
+    def __init__(self, looker_instance_data: Optional[LookerInstanceData]):
```

Comment:
biasing towards being explicit here so that we are always consciously providing the instance data, if available, and only passing `None` if we are in a situation where we don't have access to it (the builder fn, for now) - want to avoid a situation where we start implicitly passing `None` without realizing it by omitting the param

Reply:
That's fair, thanks for clarifying!